cogs5e/lookup.py

```diff
@@ -185,25 +185,25 @@ async def _feat(self, ctx, result: gamedata.feat):
         await destination.send(embed=embed)
 
     # ==== races / racefeats ====
-    @commands.command()
+    @commands.command(aliases=["speciesfeat"])
```

Comment:
I'd also have considered `@commands.command(name="speciesfeat", aliases=["racefeat"])` and the such, if you wanted this to be species forward. With how this is, it will appear as `!racefeat` in `!help`, and `![racefeat|speciesfeat]` in `!help racefeat`

![image](https://github.com/user-attachments/assets/30909e1c-7b5b-434d-9084-76f26d35c701)
![image](https://github.com/user-attachments/assets/402393d2-55fa-4f60-b672-ff8c7e1decd2)